airbyte-integrations/connector-templates/source-python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM airbyte/integration-base-python:dev
+FROM airbyte/integration-base-python:0.1.0
```

Comment:
I wonder if we should `s/integration/connector` at the same time.

Reply:
Good idea, will do in the downstream PR